src/serialize_core.hh

```diff
@@ -79,25 +79,23 @@ template<typename T1, typename T2> struct SerializeClassVersion<std::pair<T1, T2
  *
  * The serialize_as_enum class has the following members:
  *  - static bool value
- *      True iff this type must be serialized as an enum.
+ *      True if this type must be serialized as an enum.
```

Comment:
This was not a typo, we use [iff](https://www.merriam-webster.com/dictionary/iff) as an abbreviation of "if-and-only-if".